Gemfile

```diff
@@ -30,6 +30,9 @@ group :development, :test do
   gem 'web-console', '~> 2.0'
   gem 'spring'
   gem 'minitest'
+  gem "minitest-rails", "~> 2.2.0"
```

Comment:
Yes @sivagollapalli. Will remove it later. Added for generating tests files for existing models and controller.